pkg/gowork/system.go

```diff
@@ -0,0 +1,54 @@
+package gowork
```

Comment:
Do we no longer require copyright headers on files?

Reply:
I haven't added them for 2y. There's a LICENSE on root repo level.

From: https://softwareengineering.stackexchange.com/a/125841

> _Your software is automatically covered under copyright, as agreed in international law._

I think adding the license header was a Google policy. I think it is pointless and outdated. The copyright law is active, no matter if you add a file header or not. Otherwise, it would be pretty silly law. The GPLv3 strongly encourage people to attach the header, but I doubt it's grounded in any legal stuff.